src/Assignment3/search/SearchEx6.kt

```diff
@@ -0,0 +1,23 @@
+package Assignment3.search
+
+class SearchEx6 {
+
+    // This function checks whether arr2 is a subset of arr1 or not. arr1 and arr2 have only distinct integers.
+    fun arrayIsSubset(arr1: IntArray, arr2: IntArray) : Boolean {
+        if (arr2.isEmpty()) return true // An empty set is a subset of any other set
+
```

Comment:
one small improvement here is to use the relatively smaller array to convert to Set if one of the arrays is significantly larger than another.